static/css/components/buttonsAndLinks.less

```diff
@@ -113,3 +113,4 @@ button {
 @import "components/buttonLink.less";
 @import "components/buttonGhost.less";
 @import "components/buttonBtn.less";
+@import "components/searchbox.less";
```

Comment:
This doesn't make sense in `buttonsAndLinks` anymore; let's put it in `page-user`, that's the css entry file used by the search page.